src/translations/en.json

```diff
@@ -174,7 +174,7 @@
 	  "subtitle":"Help to fund the development and pay the bills",
 	  "webwalletBlock":{
 		"title":"Webwallet",
-		"content":"This Web wallet is developed by Gnock on his free time.<br/>If you want to help the development, help to pay for the servers, and like this app, please consider a donation: <br/><br/>"
```

Comment:
This should stay as gnock as the donation address here is his

Reply:
ok got it